mind_matter_api/app.py

```
@@ -43,6 +43,7 @@ def create_app(config_object="mind_matter_api.settings"):
     register_shellcontext(app)
     register_commands(app)
     register_routes(app)
+    print(f'app.url_map')
```

Ruff (0.8.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)